src/examples/dropdowns/combobox/Default.tsx

```diff
@@ -0,0 +1,103 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useMemo, useState, useEffect } from 'react';
+import debounce from 'lodash.debounce';
+import { Item, Menu, Label, Field, Dropdown, Combobox } from '@zendeskgarden/react-dropdowns';
+import { Row, Col } from '@zendeskgarden/react-grid';
+import { ReactComponent as SearchIcon } from '@zendeskgarden/svg-icons/src/16/search-stroke.svg';
+import { Dots } from '@zendeskgarden/react-loaders';
+
+const options = [
+  'Asparagus',
+  'Brussel sprouts',
+  'Cauliflower',
+  'Garlic',
+  'Jerusalem artichoke',
+  'Kale',
+  'Lettuce',
+  'Onion',
+  'Mushroom',
+  'Potato',
+  'Radish',
+  'Spinach',
+  'Tomato',
+  'Yam',
+  'Zucchini'
+];
+
+const Example = () => {
+  const [isLoading, setLoading] = useState(false);
+  const [selectedItem, setSelectedItem] = useState('');
+  const [inputValue, setInputValue] = useState('');
+  const [matchingOptions, setMatchingOptions] = useState<string[]>([]);
+
+  /**
+   * Debounce filtering
+   */
+  const filterMatchingOptions = useMemo(
+    () =>
+      debounce((value: string) => {
+        if (value.length > 0) {
+          setMatchingOptions(
+            options.filter(
+              option => option.trim().toLowerCase().indexOf(value.trim().toLowerCase()) !== -1
+            ) || []
+          );
+        } else {
+          setMatchingOptions([]);
+        }
+        setLoading(false);
+      }, 750),
+    [setMatchingOptions]
+  );
+
+  useEffect(() => {
+    if (inputValue) {
+      setLoading(true);
+    }
+
+    filterMatchingOptions(inputValue);
+  }, [filterMatchingOptions, inputValue]);
+
+  return (
+    <Row justifyContent="center">
+      <Col sm={5}>
+        <Dropdown
+          inputValue={inputValue}
+          selectedItem={selectedItem}
+          onSelect={item => setSelectedItem(item)}
+          onInputValueChange={value => setInputValue(value)}
+        >
+          <Field>
+            <Label>Choose a vegetable</Label>
+            <Combobox
+              start={<SearchIcon />}
+              end={isLoading ? <Dots delayMS={0} /> : null}
```

Comment:
I'm still seeing the jitter on hover after initial load. Have you looked into removing these lines? The component was never tested in conjunction with the `<Dots>` loader. In general, the intent of these start/end slots is not for containing loader assets, but rather for icons.  We don't currently have examples on the site, but in the past the input-loader pattern has been expressed in a different manner altogether – see the "Loading items..." treatment under https://garden.zendesk.com/components/multiselect#default. This would also mitigate the false "No matches found" message that briefly appears prior to a successful search.

I would recommend removing this `end` throughout and seeing if that alleviates the jitter-on-hover.

Reply:
I see the jitter on hover now.. Hmm, both the loaders and icons are technically SVG elements so I don't immediately suspect the `Dots` component, I'll test it out without the `Dots` component and see it it helps.